### DISCUSSION TITLE
Fatal error, resolved by clearing Pod Cache

### DISCUSSION THREAD

**Discussion by jessicaperry** [pods-framework/pods#6622] [Q&A]
Re-posting this from forum to here:

The following error keeps showing up intermittently on my site (for some pages, but not others, and never for the same page).

Fatal error: is_callable(): The script tried to execute a method or access a property of an incomplete object. Please ensure that the class definition "PodsField_Pick" of the object you are trying to operate on was loaded _before_ unserialize() gets called or provide a __autoload() function to load the class definition in /home/deer/public_html/wp-content/plugins/pods/classes/fields/pick.php on line 1377

If I go and clear the Pod Cache (Pods Admin > Settings > Clear Pods Cache), it fixes the issue. However, several days later, the error shows up on another page.

Most recently, it showed up on: http://wyocoopunit.org/people/jenny-jones

Settings below. No, I don't have a copy of the site, unfortunately.

Debug Information

WordPress Version: 4.4.2

PHP Version: 5.4.45

MySQL Version: 5.5.45

Server Software: Apache/2.4.16

Your User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:44.0) Gecko/20100101 Firefox/44.0

Session Save Path: /tmp

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

```
Apc: Yes
Memcached: No
Redis: No
```

Object Cache:

```
Apc: Yes
Apcu: No
Memcache: No
Memcached: No
Redis: No
```

WPDB Prefix: wp_

WP Multisite Mode: No

WP Memory Limit: 128M

Pods Network-Wide Activated: No

Pods Install Location: /home/deer/public_html/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: Zealth

Currently Active Plugins:

```
Adminimize: 1.10.0
Admin Menu Editor: 1.5
Block Specific Plugin Updates: 2.1
BulletProof Security: .53.2
Editorial Access Manager: 0.3.1
Error Log Monitor: 1.3.1
EWWW Image Optimizer: 2.5.9
Pods - Custom Content Types and Fields: 2.6.1
UpdraftPlus - Backup/Restore: 1.11.27
User Role Editor: 4.23.2
W3 Total Cache: 0.9.4.1
Wordfence Security: 6.0.24
WP-Optimize: 1.8.9.10
WP Canvas - Gallery: 1.40
WPFront User Role Editor Personal Pro: 2.12.5
```

Thanks for any insight you can provide!

**Comment by jimtrue** [comment]
I doubt it's the version of PHP, but that might be an issue. Can you look into upgrading your PHP on this particular web server?

**Comment by GaryJones** [comment]
@jessicaperry Did you get this resolved?
